templates/profile_view/profile.html

```diff
@@ -8,26 +8,56 @@
     <link rel="stylesheet" href="{% static 'profile_view/style.css' %}">
 </head>
 <body>
+    <header>
+        <nav>
+            <ul class="operation">
+                <li><a href="#">Explorer</a></li>
+                <li><a href="#">Notification</a></li>
+                <li><a href="/create-section/">+ Add Section</a></li>
+                {% if isAdmin%}
+                    <li><a href="/create-course/">+ Add Course</a></li>
+                    <li><a href="/create-user/">Create User</a></li>
```

Comment:
add + to create user to make it same as add course

Reply:
Got